docker-compose.yml

```
@@ -29,6 +29,19 @@ services:
       - ptolemy
     depends_on:
       - postgres
+  ui:
+    container_name: ui
+    build:
+      context: ptolemy-ui/
+      dockerfile: Dockerfile
+    ports:
+      - "3000:80"
+    networks:
+      - ptolemy
+    environment:
+      VITE_PTOLEMY_API: http://api:8000
```

should we also include the docs link?

```suggestion
      VITE_PTOLEMY_API: http://api:8000
      VITE_PTOLEMY_DOCS: http://localhost:8080
```

also realize the Feedback link points to a fake email; where do you want me to point to?

---

@pachewise raz@besaleli.io